build/Dockerfile.j2

```diff
@@ -1,3 +1,9 @@
+{# These ARGs are needed because they go out of scope at the end of the build stage where it was defined #}
+{# https://docs.docker.com/engine/reference/builder/#scope #}
+{% for argument in arguments %}
```

Comment:
This is needed to ensure that the `ARG KICS_VERSION` gets fed down to the `FROM` statement for the `Dockerfile.kics`